index.html

```diff
@@ -10,5 +10,143 @@
+        <nav>
+          <ul class="flex text-xs font-medium items-center">
+            <li class="border-r-2 p-x flex-1 px-4"><a href=""><span class="font-bold">NEWSLETTER</span></a></li>
+            <li class="px-4"><a href="">HOME</a></li>
+            <li class="px-4"><a href="">ABOUT</a></li>
+            <li class="px-4"><a href="">FAQ</a></li>
+            <li class="px-4"><a href="">CONTACT US</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </div>
 
-</html>
+  <!--slider div start here-->
+
+  <div class="bg-white">
+    <div class="w-10/12 my-0 mx-auto flex justify-between items-center h-24">
+      <div class=""><img src="./images/logo_black.png" class="h-6"></div>
+      <div class="flex gap-x-4">
+        <nav class="">
+          <ul class="flex gap-x-4">
+            <li>
+              <href="" class="text-[#b1b1b2]">HOME</a>
+            </li>
+            <li>
+              <href="" class="text-Neutral-950">PAGES</a>
+            </li>
+            <li>
+              <href="" class="text-Neutral-950">BLOG</a>
+            </li>
+            <li>
+              <href="" class="text-Neutral-950">SHOP</a>
+            </li>
+            <li>
+              <href="" class="text-Neutral-950">ELEMENTS</a>
+            </li>
+            <li>
+              <href="" class="text-Neutral-950">CATEGORIES</a>
+            </li>
+          </ul>
+        </nav>
+        <nav class="">
+          <ul class="flex">
+            <li class="pr-3.5"><a href=""><i class="fa-brands fa-facebook-f text-[#b1b1b2]"></i></a></li>
+            <li class="pr-3.5"><a href=""><i class="fa-brands fa-instagram text-[#b1b1b2]"></i></a></li>
+            <li class="pr-3.5"><a href=""><i class="fa-brands fa-pinterest text-[#b1b1b2]"></i></a></li>
+            <li class="pr-3.5"><a href=""><i class="fa-regular fa-square-plus text-[#b1b1b2]"></i></a></li>
+            <li class="pr-3.5"><a href=""><i class="fa-brands fa-square-vimeo text-[#b1b1b2]"></i></a></li>
+            <li class="pr-3.5 border-r-2"><a href=""><i class="fa-solid fa-rss text-Neutral-950"></i></a></li>
+            <li class="px-3.5"><a href=""><i class="fa-regular fa-envelope text-Neutral-950"></i></a></li>
+            <li class="pr-3.5"><a href=""><i class="fa-solid fa-magnifying-glass text-Neutral-950"></i></a></li>
+            <li class="pr-3.5"><a href=""><i class="fa-solid fa-bars text-Neutral-950"></i></a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </div>
+
+  <div class="w-10/12 mt-7 mx-auto flex">
+    <div class="pt-6 px-10 pb-5 bg-white flex-1">
+      <div class="m-2.5">
+        <h6><span class="text-[#b1b1b2]">UPDATE</span></h6>
+      </div>
+      <div class="">
+        <h2 class="text-4xl font-bold"><a href="">Just How Important Decoration is For Your Office</a></h2>
+      </div>
+      <div class="pt-[28%] align-left">
+        <div class="w-[53px] h-[53px]  bg-[#b0b0b0] flex justify-center items-center rounded-full">
```

Comment:
```suggestion
        <div class="w-[53px] h-[53px]  bg-[#b0b0b0] flex justify-center items-center rounded-full">
```
<div class="w-[53px] h-[53px] bg-[#b0b0b0] flex justify-center items-center rounded-full">